src/config.ts

```diff
@@ -33,7 +33,10 @@ function onErrorRetry(
     return
   }
 
-  if (config.errorRetryCount && opts.retryCount > config.errorRetryCount) {
+  if (
+    config.errorRetryCount != null &&
```

Comment:
```suggestion
    typeof config.errorRetryCount === 'number' &&
```

We should have that check no matter the option is set to `null` or `undefined`, so I suggest to test if it's a number.